models/addressable.go

```diff
@@ -184,9 +184,10 @@ func (a *Addressable) UnmarshalJSON(data []byte) error {
 
 // Validate satisfies the Validator interface
 func (a Addressable) Validate() (bool, error) {
+	fmt.Println(fmt.Sprintf("Addressable.Validate isValidated=%v", a.isValidated))
```

Comment:
If I'm not wrong, this line should be removed

Reply:
Oh, good catch! Yes, this was part of my debugging when I was trying to understand why tests were failing. See fourth bullet in PR description.